modules/profile/variables.tf

```diff
@@ -0,0 +1,41 @@
+########################################################################################################################
+# Input variables
+########################################################################################################################
+
+variable "instance_id" {
+  type        = string
+  description = "The ID of the SCC instance in a particular region."
+}
+
+variable "profile_name" {
+  type        = string
+  description = "The profile name. Constraints: The maximum length is `64` characters. The minimum length is `2` characters. The value must match regular expression `/[A-Za-z0-9]+/`."
+}
+
+variable "profile_description" {
+  type        = string
+  description = "The profile description. Constraints: The maximum length is `256` characters. The minimum length is `2` characters. The value must match regular expression `/[A-Za-z0-9]+/`."
+}
+
+variable "controls" {
+  type = list(object({
+    control_library_name    = optional(string)
+    control_library_version = optional(string)
+    control_list            = optional(list(string))
```

Comment:
I suggest you rename this to `control_name_list`.

Am I correct in thinking that if nothing is passed for this, all the controls for the control library will be added?
Lets make sure the description is clear here

Reply:
No controls are added. Should we not add a flag instead if the user wants every control from a ctrl library? The former might be confusing.

Reply:
add a flag to add all controls